### ISSUE TITLE
El addon ha dejado de funcionar

### ISSUE THREAD

**Issue by susobaco** [josecurioso/service.subtitles.tusubtitulo#5]
Hola, el addon ha dejado de funcionar, simplemente no encuentra ningún subtítulo. 

Adjunto log:
[log.txt](https://github.com/josecurioso2/service.subtitles.tusubtitulo/files/3044815/log.txt)

Kodi Leia sobre Trisquel Flidias (Ubuntu xenial)

Salud!

**Comment by susobaco** [comment]
No hay novedades, no?
Gracias.

**Comment by josecurioso** [comment]
No, la verdad es que dejé de utilizarlo yo mismo hace tiempo. Ahora me los bajo de [subtitulamos.tv](https://www.subtitulamos.tv/) y en sus foros hay alguien que ha hecho un addon para Kodi. Te lo dejo aquí por si te sirve:
[https://foro.subtitulamos.tv/viewtopic.php?f=10&t=1103](https://foro.subtitulamos.tv/viewtopic.php?f=10&t=1103)
[https://github.com/julianyus/subtitulamostv](https://github.com/julianyus/subtitulamostv)

**Comment by susobaco** [comment]
Gracias.
